openmdao/docs/features/recording/case_reader.rst

```diff
@@ -0,0 +1,161 @@
+along the path. It follows a pattern of repeated pairs of
+
+    - object name ( problem, driver, system, or solver )
+    - iteration count
+
+These are separated by the :code:`|` character.
+
+So in the given example, the case is:
+
+    - from rank 0
+    - the first iteration of the driver, `ScipyOptimize_SLSQP`
+    - the first execution of the `root` system which is the top-level model
+
+Getting Names of the Cases
+--------------------------
+
+The :code:`list_cases` method returns the names of the cases in the order in which
+the cases were executed. You can optionally request cases only from a specific :code:`source`.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_cases
+    :noindex:
+
+.. _list_cases_args:
+
+There are two optional arguments to the :code:`list_cases` method that effect what is returned.
+
+    - :code:`recurse`: causes the returned value to include child cases.
+
+    - :code:`flat`: works in conjunction with the :code:`recurse` argument to determine if the returned
+      results are in the form of a list or nested dict. If recurse=True, flat=False, and there are child cases, then
+      the returned value is a nested ordered dict. Otherwise, it is a list.
+
+
+Getting Access to Cases Using get_cases Method
+----------------------------------------------
+
+Getting information from the cases is a two-step process. First, you need to get access to the Case object and then
+you can call a variety of methods on the Case object to get values from it. The second step is described on the
+:ref:`Accessing Recorded Data<reading_case_data>` page.
+
+There are two methods used to get access to :code:`Cases`:
+
+    - :code:`get_cases`
+    - :code:`get_case`
+
+
+Getting Access to Cases Using get_cases Method
+----------------------------------------------
+
+The :code:`get_cases` method provides a quick and easy way to iterate over all the cases.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.get_cases
+    :noindex:
+
+The method :code:`get_cases` is similar to the :code:`list_cases` method in that it has the two optional arguments
+:code:`recurse` and :code:`flat` to control what is returned and the data structure returned. See
+:ref:`explanation of the list_cases args<list_cases_args>`.
+
+Here is an example of its usage.
+
+.. embed-code::
+    openmdao.recorders.tests.test_sqlite_reader.TestFeatureSqliteReader.test_feature_get_cases
+    :layout: code, output
+
+Getting Access to the Case Values Using get_case Method
+-------------------------------------------------------
+
+The :code:`get_case` method returns a :code:`Case` object given a case name.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.get_case
+    :noindex:
+
+If after getting the list of case names using the :code:`list_cases` method, you determine the case you want information
+from, you can use the :code:`get_case` method to get that case.
```

Comment:
Suggest simplifying this sentence.. something like:

"You can use the `get_case` method to get a specific case from the list of case names returned by `list_cases`."

Reply:
Thanks for all the comments. Made the suggested fixes and pushed up a new version